go/driver_eg.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "C"
+import (
+	"fmt"
+	"os"
+	"typedb_driver/go_wrapper"
+)
+
+func openCoreFunc() {
+	pwd, err := os.Getwd()
+	if err != nil {
+		fmt.Println(err)
```

Comment:
The formatting looks a bit weird to me with the amount of spaces for tabs, but I'm not sure about the `go` code conventions. Let's learn it and start following everywhere in your code.